Sources/Hyperwallet.swift

```diff
@@ -248,6 +248,31 @@ public final class Hyperwallet {
                                     completionHandler: completion)
     }
 
+    /// Commits the `HyperwalletTransfer` linked to the transfer method token specified.
+    ///
+    /// The `completion: @escaping (HyperwalletStatusTransition?, HyperwalletErrorType?) -> Void` that is passed in to
+    /// this method invocation will receive the successful response(HyperwalletStatusTransition) or
+    /// error(HyperwalletErrorType) from processing the request.
+    ///
+    /// This function will request a new authentication token via `HyperwalletAuthenticationTokenProvider`
+    /// if the current one is expired or is about to expire.
+    ///
+    /// - Parameters:
+    ///   - transferToken: the Hyperwallet specific unique identifier for the `HyperwalletTransfer`
+    ///                    being commited
+    ///   - notes: a note regarding the status change
+    ///   - completion: the callback handler of responses from the Hyperwallet platform
+    public func commitTransfer(transferToken: String,
```

Comment:
rename it to scheduleTransfer to keep it consistent with the UI

Reply:
Renamed